test/mocks/carbonio-shell-ui.tsx

```diff
@@ -31,6 +31,8 @@ export const useUserAccount: jest.Mock<ReturnType<typeof shell.useUserAccount>>
 export const useUserAccounts: jest.Mock<ReturnType<typeof shell.useUserAccounts>> = jest.fn(
 	() => mockedAccounts
 );
+export const getFolder: jest.Mock<ReturnType<typeof shell.getFolder>> = jest.fn();
```

Comment:
The fact that this mock is needed is an indicator that the wrong getter is used: the whole folders management from _Shell_ will be soon removed.
Probably the Commons submodule `getFolder` has to be used instead